dts/bindings/ethernet/qca,ar8031.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: Qualcomm Atheros AR8031 Ethernet PHY
+
+compatible: "qca,ar8031"
+
+include: ethernet-phy.yaml
+
+properties:
+  eee:
```

Comment:
probably a better name for the property would be `eee-en` at least (enable)

Reply:
Updated, thanks.